api-docs/swagger.yml

```diff
@@ -402,6 +402,9 @@ definitions:
           cpu:
             type: string
             example: "0.01"
+      readOnlyRootFilesystem:
```

Comment:
We can add `readOnlyRootFilesystem` before finishing this `WIP` PR.

@ewilde Please could you or your colleague raise a PR for the `readOnlyRootFilesystem` part of this?

Reply:
@alexellis i'm no longer working on this PR, so going to close. @ameier38 is picking this up